src/services/SendATFReport.ts

```diff
@@ -25,25 +25,29 @@ class SendATFReport {
    * @param generationServiceResponse - The response from the ATF generation service
    * @param visit - Data about the current visit
    */
-  public async sendATFReport(generationServiceResponse: any, visit: any): Promise<any> {
+  public async sendATFReport(generationServiceResponse: any, visit: any) {
     // Add testResults and waitActivities in a common list and sort it by startTime
     const activitiesList = this.computeActivitiesList(generationServiceResponse.testResults, generationServiceResponse.waitActivities);
 
     const response = await this.testStationsService.getTestStationEmail(visit.testStationPNumber);
+    console.debug(`response from get test station email: ${JSON.stringify(response)}`);
```

Comment:
do we need to be concerned about logging emails here? Are any of the test station emails PII that we need to be concerned about?